### ISSUE TITLE
Usability of Anaconda groups on CLI

### ISSUE THREAD

**Issue by rafaelhbarros** [Anaconda-Platform/anaconda-client#290]
It's virtually impossible for someone who doesn't have access to code to use `anaconda groups` command. There is zero docs and it's like pulling teeth with a rusted pair of pliers.

**Comment by rafaelhbarros** [comment]
example:

```
parser.add_argument('action', help='asdf',
                        choices=['add', 'show', 'members', 'add_member', 'remove_member',
                                 'packages', 'add_package', 'remove_package'], nargs='?')
```

help='asdf' is a bit out there.

**Comment by rafaelhbarros** [comment]
It only matches lower case, and this is roughly the spec: org/:group/:member

**Comment by stephenakearns** [comment]
cc: @malev

**Comment by dsludwig** [comment]
Fixed, due to be released in 1.4.1
